apis/v1beta1/vspherecluster_types.go

```diff
@@ -59,6 +66,9 @@ type VSphereClusterStatus struct {
 
 	// FailureDomains is a list of failure domain objects synced from the infrastructure provider.
 	FailureDomains clusterv1.FailureDomains `json:"failureDomains,omitempty"`
+
+	// VCenterVersion defines the version of the vCenter server defined in the spec.
+	VCenterVersion VCenterVersion `json:"vCenterVersion,omitempty"`
```

Comment:
this should probably be a string. This would align on how we treat version in CAPI/CAP*

Reply:
This points to a string.